pandas/util/_depr_module.py

```diff
@@ -34,7 +36,7 @@ def __init__(self, deprmod, deprmodto=None, removals=None, moved=None):
         # For introspection purposes.
         self.self_dir = frozenset(dir(type(self)))
 
-    def __dir__(self):
+    def __dir__(self) -> List[str]:
```

Comment:
Minor but I think should match the stdlib annotation here using `Iterable`:

https://github.com/python/typeshed/blob/5fdd6ad1a59d9a7e44979866b3b9acabfee64a90/stdlib/2and3/builtins.pyi#L67

Reply:
Makes sense